config/locales/forms/check_your_answers_forms/en.yml

```diff
@@ -43,12 +43,12 @@ en:
           subheading_address: Contact address
         farming:
           subheading: Farming information
-          is_a_farm:
-            true_html: "the operater <span class=\"strong\">is</span> a farmer"
-            false_html: "the operater <span class=\"strong\">is not</span> a farmer"
           on_a_farm:
             true_html: "the operation <span class=\"strong\">will</span> take place on a farm"
             false_html: "the operation <span class=\"strong\">will not</span> take place on a farm"
+          is_a_farmer:
+            true_html: "the operater <span class=\"strong\">is</span> a farmer"
```

Comment:
I know this was an old issue, but the spelling is actually `operator`

Reply:
🤦‍♂️